vm_supervisor/conf.py

```diff
@@ -105,6 +105,11 @@ class Settings(BaseSettings):
     MAX_PROGRAM_ARCHIVE_SIZE = 10_000_000  # 10 MB
     MAX_DATA_ARCHIVE_SIZE = 10_000_000  # 10 MB
 
+    # hashlib.sha256(b"secret-token").hexdigest()
+    ALLOCATION_TOKEN_HASH = (
+        "151ba92f2eb90bce67e912af2f7a5c17d8654b3d29895b042107ea312a7eebda"
```

Comment:
Same question you had for the RabbitMQ username/password on CCNs: how de we guarantee that node operators do not blindly use the default value?

Reply:
This is the hash of a secret token stored on the scheduler, not a password.

Reply:
It is also temporary, and will be deprecated when switching to consensus scheduling with P2P.

Reply:
Ah yeah my bad, a bit tired. Ok for me.